ang/test/civicase-base/providers/case-type-category.provider.spec.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-env jasmine */
+((_, angular) => {
+  describe('CaseTypeCategory Provider', () => {
+    let CaseTypeCategory;
+
+    describe('when all case type categories are requested', () => {
+      beforeEach(() => {
+        module('civicase-base');
+        injectDependencies();
+      });
+
+      it('returns all available case type categories', () => {
+        expect(CaseTypeCategory.getAll()).toEqual(CRM['civicase-base'].caseTypeCategories);
```

Comment:
To be honest not sure if its worth it. We have used direct function calls in `it` statements in many places. And not sure adding another variable to save the `returnValue` will be much beneficial. It would make sense if the function call statement is really large or something, but in this case it seems fine to me.